3drepo/3drepo.io
# ISSUE #3727 - V5 Teamspace user list MVP

This fixes #3727

#### Description
Adds V5 style to the teamspace user settings
This also fixes ISSUE #3635

#### Test cases

Make sure you check each of the following:
- Add user Popover
- Add user popover when trying to add a user that doesn't exist
- Invite user modal empty
- Invite user modal with project selected
- Invitees list filled
(click here) ![image](https://user-images.githubusercontent.com/62481250/204585502-ef93bdb4-55a8-4748-a40b-f67d6608fb2a.png)
- Invitees list empty
- Delete teamspace admin confirmation modal
